contracts/PoolManager.sol

```diff
@@ -59,7 +59,7 @@ contract PoolManager is IPoolManager, Owned, NoDelegateCall, ERC1155, IERC1155Re
     }
 
     /// @inheritdoc IPoolManager
-    function getSlot0(bytes32 poolId)
+    function getSlot0(PoolId poolId)
```

Comment:
we can rename `poolId` to `id` here to be more consistent with `PoolKey key`

Reply:
Only in functions or in events as well?